src/Text/Text.stories.tsx

```diff
@@ -2,32 +2,33 @@ import React from 'react'
 import styled from 'styled-components'
 import { Text, TextProps } from './Text'
 import { Box } from '../Box'
+import { Typo } from './Text'
 
 export default {
   title: 'Text',
   component: Text,
 }
 
 const typos = [
-  'header-large',
```

Comment:
wondering if we should keep displaying the deprecated typos in a separate story?

Reply:
I'll add a new collection